### PR TITLE
[Monitor] Metric Alerts commands

### PR DESCRIPTION
**Pull request by tjprescott**
Add support for near-realtime metric alerts. Deprecates and hides the old `monitor alter` metric alert commands.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by tjprescott** [comment]
@troydai ping.

File: src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/operations/metric_alert.py [hunk 1]
```diff
@@ -2,10 +2,99 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
+from __future__ import print_function
```
Comment (troydai) [inline src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/operations/metric_alert.py]: Ugr, what is this for? Was `print` used in this file?

File: src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/actions.py [hunk 2]
```diff
@@ -5,6 +5,8 @@
 
 import argparse
 
+import antlr4
```
Comment (troydai) [inline src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/actions.py]: Was it brought in by the monitor module?

Comment (tjprescott) [reply src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/actions.py]: Yes. This is needed for the grammar parser.

File: src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/operations/metric_alert.py (hunk 1 above)
Comment (tjprescott) [reply src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/operations/metric_alert.py]: Mmmm...yeah. I can remove it.

**Pull request merged**